test/addons/rook-cluster/kustomization.yaml

```diff
@@ -3,9 +3,11 @@
 
 # yamllint disable rule:line-length
 ---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
You should mention in the commit message that you added apiVersion and kind to the modified kustomization files.

Looks like a good change, but unrelated to the patchesJson6902 change, and also partial, since we have many other kustomization files without this. I think it is better to separate this change from the patchesJson6902 PR, and post another PR modifying all kustomization files under test/.

We also have other kustomization files under config, which may need the same change. If you decide to change them it should be in a different commit since these files are not part of the tests, but examples that actual users may apply on clusters.